### PR TITLE
Added Popup modal

### PR DESCRIPTION
**Pull request by nxn404-n**
### In this branch

1. Added JavaScript data file.
2. Added new popup module.

### CODE REVIEW

**Comment by huseyinbiyik** [comment]
Hi @nxn404-n,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

Alt Text

## Invalid Code Review Request

You have submitted a project, that has no contribution from your pair programming partner. If your partner is missing and you contacted student success and they let you complete this project solo, please share the image of the email.
Screenshot 2023-02-02 at 7 46 39 PM


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!�

�👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: js/modal.js [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+     
+         <img src="${data.separator}" class="sep">
+            <div class="modal-hid__content__lower">
+               <button class="sm-btn lowerBtn">See live <img src="${data.liveLogo}"></button>
+              <button class="sm-btn lowerBtn">See Source <img src="${data.sourceLogo}"></button>
+            </div>
+        </div>
+    </div>
+</div>
+  `;
```
Comment (Ibizugbe) [inline js/modal.js]: Nice job implementing the required functionalities of the popup window👍 but I think we can still make our popup window look more like the design for both desktop and mobile👍  don't forget, it is a [requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/8_popup.md) that we stick to the given [design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=zJuNtmSox7xPcTIq-0) as much as possible in this milestone😉

your popup (desktop)
image


design(desktop)
Screenshot from 2023-02-02 18-26-04

your project (mobile)
image

design (mobile)
image

kindly take note of the appearance, spacings, and image width and make the necessary changes👍

**Review by Ibizugbe — CHANGES_REQUESTED** [review]
Hi Team, 

68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f4a713832345239334a734c775a436169534c2f67697068792e676966


Good job so far!👍
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights✅
- [x] The JavaScript code that implements an array of objects to store JavaScript objects containing the information for each project has be created👍.
- [x] Modal popup and projects data is populated with the data from the projects array👍.
- [x] Linters are green👍.


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Felix45 — APPROVED** [review]
# Approved :green_circle: 
Hi @nxn404-n , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights  :heavy_check_mark: 
- [ ] No linter errors :white_check_mark: 
- [ ] GitHub flow is used correctly :white_check_mark: 
- [ ] Repository has a professional readme file :white_check_mark: 
- [ ] Implemented changes from the previous reviewer :white_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**